docs/_includes/intrinsic_attributes.adoc

```diff
@@ -0,0 +1,467 @@
+|Sets the text used to label EXAMPLES.
+|Example
+|Any (value not used)
+|-
+
+|experimental
+|Enable experimental extensions. 
+Emphasises that they are subject to change and may even be removed in the next version.
+Currently only applies to the interface macros.
+|(not set)
+|Any (value not used)
+|<<user-interface-macros>>
+
+|figure-caption
+|Sets the text used to label figures.
+|Figure
+|Any
+|<<images>>
+
+|hardbreaks
+|Preserve hard line breaks in the input.
+|(not set)
+|Any (value not used)
+|<<line-breaks>>
+
+|icons
+|Chooses icons instead of text for admonitions.
+|(not set)
+|font
+|<<admonition-icons>>
+
+|iconsdir
+|Where admonition icons are stored.
+|./images/icons
+|Path
+|<<icons>>
+
+|idprefix
+|Prefix for auto-generated section IDs.
+|(not set)
+|Any
+|<<auto-generated-ids>>
+
+|imagesdir
+|Where image files are stored.
+|(not set) in my tests, ./images in text
+|Path
+|<<images>>
+
+|important-caption
+|Sets the text used to label IMPORTANT admonitions, if icons are not used.
+|Important
+|Any
+|<<admonition>>
+
+|indent
+|Normalize block indentation in code listings.
+|(not set)
+|Any (value not used)
+|<<normalize-block-indentation>>
+
+|lang
+|Set the `lang` attribute in the Docbook backend.
+|en
+|Any valid XML country code
+|
+
+|last-update-label
+|Text label for the "last updated" time in the footer.
+Unsetting it will remove the label and time from the footer.
+|Last updated
+|Any
+|<<footer-docinfo-files>>
+
+|linkcss
+|link to the stylesheet instead of embedding it.
+|(not set)
+|Any (value not used)
+|<<styling-the-html-with-css>>
+
+|mantitle
+|Metadata for manpage output.
+|(not set)
+|Any
+|<<man-pages>>
+
+|manvolnum
+|Metadata for manpage output.
+|(not set)
+|Any
+|<<man-pages>>
+
+|manname
+|Metadata for manpage output.
+|(not set)
+|Any
+|<<man-pages>>
+
+|manpurpose
+|Metadata for manpage output.
+|(not set)
+|Any
+|<<man-pages>>
+
+|manname-title
+|Label for the program name liner in manpage outputs?
+|NAME
+|Any
+|?
+
+|max-include-depth
+|Not normally used. Should it be exposed?
+|64
+|-
+|-
+
+|menu
+|Used by Deck-JS backend.
+|(not set)
+|Any (value not used)
+|<<deckjs-backend-attributes>>
+
+|navigation
+|Used by Deck-JS backend.
+|(not set)
+|
+|<<deckjs-backend-attributes>>
+
+|no-header-footer
+|Suppresses the rendering of the header and footer.
+|(not set)
+|Any (value not used)
+|<<doc-header>> <<footer-docinfo-files>>
+
+|noheader
+|Suppresses rendering of the header.
+|(not set)
+|Any (value not used)
+|<<doc-header>>
+
+|nofooter
+|Suppresses rendering of the footer.
+|(not set)
+|Any (value not used)
+|<<footer-docinfo-files>>
+
+|Note-caption
+|Sets the text used to label NOTE admonitions, if icons are not used.
+|Note
+|Any
+|<<admonition>>
+
+|notitle
+|Toggles the display of a document’s title.
+|(not set)
+|Any (value not used)
+|<<document-title>>
+
+|nowrap
+|Use scrolling instead of wrapping in wide code listings.
+|(not set)
+|Any (value not used)
+|<<to-wrap-or-to-scroll>>
```

Comment:
nowrap is not an attribute. It's inverse, `prewrap` can be unset (using `prewrap!`) to disable wrapping.